src/components/App.js

```diff
@@ -108,7 +111,7 @@ const App = () => {
             const { code, verifier } = e.data ?? {};
             if (!code || !verifier) return;
             localStorage.removeItem(LS_SIGN_IN_POPUP);
-            dispatch(tokenHandoff(code, verifier));
+            dispatch(tokenHandoff({ code, verifier, CLIENT_ID, AUTH_CALLBACK_URL }));
```

Comment:
once again - parameters should be not in constants-case, since in tokenHandoff it isnt known if theyre truly constants